docs/en/observability/categorize-logs.asciidoc

```diff
@@ -16,16 +16,17 @@ quicker.
 Create a machine learning job to categorize log messages automatically. Machine learning will observe
 the static parts of the message, cluster similar messages, and classify them into message categories.
 
-1. Select *Categories*, and you'll be prompted to use machine learning, which will carry out the log rate categorizations.
+1. Select *Categories*. Then you'll be prompted to use machine learning, which will carry out the log rate categorizations.
 2. Choose a time range for the machine learning analysis.
```

Comment:
Do you want all of the "actionable/directive" things bolded instead of just certain things? That's typically what I do in numbered lists, but wanted to check.